integration_test/e2e_import_mnemonic_test.dart

```diff
@@ -74,3 +46,30 @@ Future<void> e2eImportMnemonicTest(WidgetTester tester) async {
   expect(currentWallet!.externalAccounts[0]!.address,
       "wit174la8pevl74hczcpfepgmt036zkmjen4hu8zzs");
 }
+
+Future<void> e2eImportMnemonicTest(WidgetTester tester) async {
+  await initializeTest(tester);
+
+  /// Assess what is on the screen
+  walletsExist = isTextOnScreen("Unlock wallet");
+  bool biometricsActive = isTextOnScreen("CANCEL");
+
+  /// Cancel the Biometrics popup for linux
+  if (walletsExist && biometricsActive && Platform.isAndroid) {
+    await tapButton(tester, "CANCEL");
+  }
+
+  if (walletsExist) {
+    /// Login Screen
+    await enterText(tester, TextFormField, password);
+    await tapButton(tester, "Unlock wallet");
+
+    /// Dashboard
+    /// Tap on the first PaddedButton on the screen, which is the identicon
+    /// and brings up the wallet list.
+    await tapButton(tester, PaddedButton, index: 0);
+    await tapButton(tester, FontAwesomeIcons.circlePlus);
+  }
+
+  await e2eImportMnemonicsUtils(tester);
```

Comment:
Shouldn't we call at the beginning the necessary instructions to set the test in the state from which we want to start testing? I think the content of mnemonicsUtils is the logic that should be placed in this test.